src/viam/sdk/components/board.cpp

```diff
@@ -23,7 +23,14 @@ API API::traits<Board>::api() {
 Board::status Board::from_proto(const viam::component::board::v1::Status& proto) {
     Board::status status;
     for (const auto& analog : proto.analogs()) {
-        status.analog_reader_values.emplace(analog.first, analog.second);
+        Board::analog_value result;
+        result.value = analog.second;
+        // The status does not contain the extra data to describe the accuracy of the reader. We
+        // fill those in with 0's instead.
```

Comment:
Should the accuracy component of `analog_reader` be optional?

Reply:
No, but there is some tension between the data obtained from reading an analog pin, and the data obtained from viewing a board's status. Part of this is resolvable by getting rid of a lot of the explicit status-reporting stuff, and it's still unclear to me whether this function is dead code. Perhaps the `Board::status` should contain just ints as the values in the map, rather than entire `Board::analog_value`s?